lib/tasks/elasticsearch.rake

```diff
@@ -1,3 +1,29 @@
 # frozen_string_literal: true
 
-require 'elasticsearch/rails/tasks/import'
+require "elasticsearch/rails/tasks/import"
+
+namespace :elasticsearch do
+  desc "create all indexes"
+  task create_all_indexes: :environment do
+    Rake::Task["provider:create_index"].invoke
+    Rake::Task["client:create_index"].invoke
+    Rake::Task["prefix:create_index"].invoke
+    Rake::Task["provider_prefix:create_index"].invoke
+    Rake::Task["client_prefix:create_index"].invoke
+    Rake::Task["doi:create_index"].invoke
+    Rake::Task["event:create_index"].invoke
+    Rake::Task["activity:create_index"].invoke
+  end
+
+  desc "delete all indexes"
```

Comment:
since all the indexes/tables are connected its a good idea to have a request to clear everything up. That way when one creates the whole setup one can be sure there are not loose ends. As we discussed in the dev meeting all the methods inside are standard rake tasks that can be triggered anyway. As mentioned there, the risk is the same.